requirements-hpu.txt

```diff
@@ -6,3 +6,4 @@ ray == 2.32.0
 triton
 pandas
 tabulate
+git+https://github.com/HabanaAI/vllm-hpu-extension.git
```

Comment:
can we fix that to a specific git hash?

Reply:
```suggestion
git+https://github.com/HabanaAI/vllm-hpu-extension@e7e0579
```

Reply:
Done, I fixed it to a newer version: https://github.com/HabanaAI/vllm-hpu-extension/commit/30ee2d1